CONTRIBUTING.md

```diff
@@ -126,11 +132,20 @@ Make sure your PR has the following
     e.g. anyone else who ate the food, or has expressed an interest in knowing the recipe
     + if your PR _requires_ a review from a particular person / people _before_ it is ready to be merged, specify this within the body of your PR
     + Only maintainers should merge the PR's - if they are slacking, please feel free to give them a nudge in the comments of the PR.
+   6. **If your recipe is yet to be cooked:** Please give the PR a `Pending Cooking` label
+  
 
 ## Instructions for maintainers
 Once a pull request has been reviewed by 3 people **who ate the meal**, as well as or including a maintainer, it can be merged. A PR should never be merged if a reviewer has requested changes and not yet re-reviewed it.
 
 If a PR is up for an unreasonable amount of time without progress, maintainers should consult the creator on whether or not it will be finished at some point, and if not, close it.
+
+If a PR has the `Pending Cooking` label attached, it can remain open indefinitely.
```

Comment:
I'm not sure about indefinitely? Maybe something the maintainers should use their best judgement on?

Reply:
I'm not sure we can trust a maintainers best judgement...